mozilla_bitbar_devicepool/test_run_manager.py

```diff
@@ -59,7 +58,7 @@ def __init__(self, wait=60, delete_bitbar_tests=False):
                     # ConnectionErrors.
                     time.sleep(5)
                     break
-                except (requests.exceptions.ConnectionError, RequestResponseError) as e:
+                except Exception as e:
                     logger.error('Failed to get tests for project %s (%s: %s).' % (project_name, e.__class__, e.message),
```

Comment:
```e.__class__``` is kind of messy...
```>>> e.__class__
<type 'exceptions.Exception'>
```
Can we do either
```
>>> e.__class__.__name__
'Exception'
>>> type(e).__name__
'Exception'
>>> 
```

ditto elsewhere below.

Reply:
Sure.